projects/slap_manipulation/src/slap_manipulation/dataloaders/robot_loader.py

```diff
@@ -262,6 +263,17 @@ def get_gripper_pose(self, trial, idx):
         ee_pose = ee_pose @ self._robot_ee_to_grasp_offset
         return ee_pose
 
+    def get_gripper_axis(self, rot_mat):
```

Comment:
maybe we need robot utils for this

Reply:
Straightforward code, made it a function so it is explicit what is happening. Going to let it remain here for now. Will move if I find a good place.

Reply:
Why not put it in Stretch Model?

Reply:
You mean `HelloStretchKinematics`?

Reply:
Or maybe `manip.py` is a better place?

Reply:
I have more of such methods which I'd like to move to a robot-specific file actually (IMO get gripper pose for example shd use robot-specific calls). I'll handle this in next PR?

Reply:
i think HelloStretchKinematics is better, nothing here that requires ROS right?

but yeah create issues for it